packages/network_info_plus/network_info_plus/example/integration_test/network_info_plus_test.dart

```diff
@@ -34,6 +67,20 @@ void main() {
       expect(_networkInfo.getWifiSubmask(), isNotNull);
       expect(_networkInfo.getWifiGatewayIP(), isNotNull);
       expect(_networkInfo.getWifiBroadcast(), isNotNull);
-    });
+    }, skip: !Platform.isAndroid);
+
+    testWidgets('test fixed network value', (WidgetTester tester) async {
+      final result = await _methodChannelNetworkInfo.getWifiSubmask();
+      expect(result, '255.255.255.0');
+      expect(
+        log,
+        <Matcher>[
+          isMethodCall(
+            'wifiSubmask',
+            arguments: null,
+          ),
+        ],
+      );
+    }, skip: !Platform.isAndroid);
```

Comment:
Move this test to `network_info_plus/test/network_info_test.dart`

And should be renamed to `test WifiSubmask MethodChannel`

Reply:
Instead of moving to `network_info_plus/test/network_info_test.dart` create a new test file in `network_info_plus/test/` called `methodchannel_test.dart` and put it there.

You can also add tests for all the methods available in the Network Info.

Also, instead of `testWidgets` it should simply be `test()`.